website/source/docs/providers/aws/d/hosted_zone.html.markdown

```diff
@@ -0,0 +1,58 @@
+---
+layout: "aws"
+page_title: "AWS: aws_hosted_zone"
+sidebar_current: "docs-aws-datasource-hosted-zone"
+description: |-
+    Provides details about a specific Hosted Zone
+---
+
+# aws\_vpc
+
+`aws_hosted_zone` provides details about a specific Hosted Zone.
+
+This resource can be useful when a module accepts a Hosted Zone name as
+an input variable and needs to, for example, add a Record Set in Route 53
```

Comment:
I would mention that it allows to find Hosted Zone ID, which is often needed, given a Hosted Zone name and certain search criteria.